### PR TITLE
Request for review from Uchenna Anya

### PR DESCRIPTION
**Pull request by uchennaanya**
The features of this project are listed below:

1. Have a nav section
   - With embedded images and videos
   - With an embedded section with grid layout
2. Have a Promo section
   - Promo Pictures
3. Apple Page nav section
   - Nav links to many other pages aligned in a flex row direction
4. Has a footer with
   - Nav links to Legal, Privacy, and Security
   - Copy right information statement

### CODE REVIEW

File: README.md
Comment (teekaytech) [inline README.md]: This section of the readme should be removed. It looks like a duplicate of what you have on line 11.

File: README.md
Comment (teekaytech) [inline README.md]: Something seems to be wrong with the arrangement of this section. Kindly observe carefully and fix it.
- For instance, you mentioned inline 47 that we should follow some steps whereas the steps are not mentioned.
- Also on line 49, if that is a header, then let it be styled like a header, then add the content. Else, you should remove it if it has no content.

File: index.html
Comment (teekaytech) [inline index.html]: Each of these items (images) should be linked. Kindly wrap it around the `<a>` tag.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @uchennaanya & @henatan99,

__I must say, you've done an amazing job so far. However, there are some changes that are required to be fixed before this project can be approved. Please check my points below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] The main background image is not rending in your live project: all I can see here is a blank background. Kindly check it out and make your background render in production.

Here's what I have here:


- [ ] The footer elements (images) should be linked. 


- [ ] I made some comments below about your readme file. Please check those comments and others, then make necessary corrections.

**[ OPTIONAL CHANGES ]**

- [ ] Your background gradient color seems to be a little bit different from the original. Kindly try to change it so it matches the original gradient color
Original page:


Your page:



**[ COMMENTS ]**

- [ ] It looks like you're working on this branch after submitting the review request. 
- To get the best and updated review, always finalize your work before requesting for review.
- Avoid pushing to the same branch after submitting your review request. You can always improve and push commits to the project after receiving your feedback from the review process.

**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

File: README.md
Comment (teekaytech) [inline README.md]: The project title has already been re-defined in line 6 below. To avoid repetition, kindly remove this in line 1.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello team :wave:

__Everything is looking good in this project, except for the fact that you still have some changes that are required to be fixed before this project can be approved. Let's check them out below:__

**[ REQUIRED CHANGES ]**

- [ ] Your main background image is still not rending in your live project. Please try to identify the cause of this bug and fix it. You can use GitHub Pages as an alternative to using githack. See how [here](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site).


**Please make the required changes and submit another review request when you are done.**

* * *

**_You can reach out to me on twitter (@ola_lere), Github(@teekaytech) or slack (@taofeekolalere )_**
__Happy Coding :smiley:__

**Review by arslanbisharat — APPROVED** [review]
## Approved 
Thank you for making all the changes requested by previous TSEs.
Please merge the branch and move forward with your curriculum.
**Arslan Bisharat**

**Pull request merged**
